### ISSUE TITLE
✨ [Mint your own NFT collection and ship a Web3 app to show them off]: <SECTION_4> - <🚀 Ship it>

### ISSUE THREAD

**Issue by okorieebube** [buildspace/buildspace-projects#432]
### What You Are Changing
*When running the Hardhat command for verifying contracts on Etherscan, the command has an argument `_totalSupply` there.
and i dont think its supposed to be there.
I was able to verify my contract only after i removed that command.
Please look into this, Thanks.*

**Comment by adilanchian** [comment]
Hey @okorieebube !

I believe this should have been merged in as a change :). Feel free to open another issue if you find anymore problems!
